lib/command.html

```diff
@@ -49,6 +49,7 @@
     font-size: 12px;
     font-family: 'Lucida Sans', 'Lucida Sans Regular', 'Lucida Grande',
       'Lucida Sans Unicode', Geneva, Verdana, sans-serif;
+    z-index: 10000;
```

Comment:
get command palette to show in front of mini bar (when there's a lot of error text, the bottom bar can cover up command palette leading it to being usable)